### PR TITLE
Improve nmea gps docs

### PR DESCRIPTION
**Pull request by hamishwillee**
This attempts to improve NMEA GPS docs, which I found a little confusing. It also moves and splits a few images. 

@Williangalvani Would appreciate your review!

### CODE REVIEW

File: en/SettingsView/General.md [hunk 1]
```diff
@@ -66,42 +67,50 @@ The settings are:
...
-- **Libre Pilot**: Autoconnect to Libre Pilot autopilot
-- **UDP**: Autoconnect to UDP
-- **RTK GPS**: Autoconnect to RTK GPS device
+- **Pixhawk:** Autoconnect to Pixhawk-series device
+- **SiK Radio:** Autoconnect to SiK (Telemetry) radio
+- **PX4 Flow:** Autoconnect to PX4Flow device
+- **Libre Pilot:** Autoconnect to Libre Pilot autopilot
+- **UDP:** Autoconnect to UDP
+- **RTK GPS:** Autoconnect to RTK GPS device
+- **NMEA GPS Device:** Autoconnect to an external GPS device to get ground station position ([see below](#nmea_gps))
```
Comment (hamishwillee) [inline en/SettingsView/General.md]: FYI adds the last field to provide forward reference to next section.

File: en/SettingsView/General.md [hunk 2]
```diff
@@ -66,42 +67,50 @@ The settings are:
...
 
-## Ground Station GPS (NMEA GPS Device) {#nmea_gps}
 
-The Ground Station GPS section is used to choose the source of GPS data that will be used for the GCS location. When there is a valid position provided by the NMEA GPS device, the location of the ground control station will be displayed on the map with a purple `Q` icon. If the GPS provides a heading, it will be indicated by the `Q` icon. The GCS location is also used to provide follow me support ([follow me](https://docs.px4.io/en/flight_modes/follow_me.html) is currently implemented for PX4 only).
+### Ground Station Location (NMEA GPS Device) {#nmea_gps}
```
Comment (hamishwillee) [inline en/SettingsView/General.md]: @Williangalvani This is section to review please.

File: en/SettingsView/General.md [hunk 3]
```diff
@@ -66,42 +67,50 @@ The settings are:
...
-3. **Networked GPS:** If you are using your USB GPS for another application, or you have some other GPS system that is not supported by the other options, you may use a UDP network port to send the GPS data to QGC. The network port for QGC to bind and listen for data must be configured by the user (see below). The data format sent to the UDP port should be in raw ASCII NMEA format.
+- USB connection:
 
-Use these menu items to configure a USB or UDP GPS:
+  ![NMEA GPS Device - Serial](../../assets/settings/general/nmea_gps_serial.jpg)
 
-- **NMEA GPS Device**: Select serial or UDP port for GPS information
-- **NMEA GPS Baudrate**: If serial port is selected, this sets the baudrate for that port
-- **NMEA Stream UDP Port**: If UDP port is selected, this is used to select the port that QGC will listen for the NMEA data (QGC binds the port as a server)
+  - **NMEA GPS Device:** *Serial*
```
Comment (hamishwillee) [inline en/SettingsView/General.md]: Is it possible to get an image from a real system for this one? @Williangalvani

Comment (Williangalvani) [reply en/SettingsView/General.md]: What do you mean? the hardware? I'm currently using an ftdi attached to a regular gps, haha.
The hardware usually looks like [this one](https://www.amazon.com/Receiver-Antenna-Gmouse-Laptop-Navigation/dp/B073P3Y48Q).
If you mean the UI, there are pictures [in the original PR in QGC].(https://github.com/mavlink/qgroundcontrol/pull/7114)

File: en/SettingsView/General.md (hunk 2 above)
Comment (Williangalvani) [reply en/SettingsView/General.md]: Looks good to me!
@jaxxzer might want to take a look too.

Comment (jaxxzer) [reply en/SettingsView/General.md]: lgtm

File: en/SettingsView/General.md [hunk 4]
```diff
@@ -66,42 +67,50 @@ The settings are:
...
+### Ground Station Location (NMEA GPS Device) {#nmea_gps}
 
-You can change the GPS source using the drop down menu in the NMEA GPS Device field. The configuration options will change based on the selected device (as shown below).
+*QGroundControl* will automatically use an internal GPS to display its own location on the map with a purple `Q` icon (if the GPS provides a heading, this will be also indicated by the icon). It may also use the GPS as a location source for *Follow Me Mode* - currently supported on [PX4 Multicopters only](https://docs.px4.io/en/flight_modes/follow_me.html).
 
-![Nmea GPS Device](../../assets/settings/settings_view_general_gps.jpg)
+If your ground station does not have an internal GPS you can instead connect an external GPS module via a serial or UDP port.
```
Comment (jaxxzer) [inline en/SettingsView/General.md]: This functionality also allows use of much higher grade GPS (not necessarily a 'module'), as an 'alternative', even if there is an internal gps. Minor thing, but could be good to frame it both ways.

File: en/SettingsView/General.md (hunk 3 above)
Comment (hamishwillee) [reply en/SettingsView/General.md]: Sorry, I thought enough context from where I added this comment. 

What I mean is that showing how to add a serial connection is not very cool when your example says "none available". There doesn't appear to be an example of this in the original PR.

image

File: en/SettingsView/General.md (hunk 4 above)
Comment (hamishwillee) [reply en/SettingsView/General.md]: Good point thank you. Updated as follows - OK @jaxxzer ?
image

File: en/SettingsView/General.md (hunk 3 above)
Comment (hamishwillee) [reply en/SettingsView/General.md]: OK, merging this now. @Williangalvani if you are able to find an appropriate image at some point we can update it. Thanks!

**Pull request merged**